engine/engine.qbs

```diff
@@ -23,8 +23,8 @@ Project {
         "../thirdparty/glfm/include",
         "../thirdparty/freetype/include",
         "../thirdparty/assimp/include",
+        "../modules/uikit/includes",
```

Comment:
I don't think engine needs this dependency. basically gui was a part of engine before. but I made separate module.